sdlib/d/gc/tcache.d

```diff
@@ -477,49 +578,6 @@ private:
 		return e.size - startIndex;
 	}
 
-	bool extend(const void[] slice, size_t size) {
```

Comment:
Moving things around make it very difficult to see what changed. Why did you move this?

Reply:
It was private. Nobody called it (except in unit tests). Basically a dead function. At first I was going to use it, but then realized it wouldn't fit my needs.

I didn't change anything, just moved it to the public section.

I can move it back if you want.

Reply:
Can you move it one PR, then we can change it?

Reply:
done